layouts/partials/navbar.html

```diff
@@ -24,6 +24,9 @@
 				{{ partial "navbar-lang-selector.html" . }}
 			</li>
 			{{ end }}
+			<li class="search-item nav-item mr-n4 mr-lg-0">
+				{{ partial "search-input.html" . }}
+			</li>
```

Comment:
We might want to show this only on wide-enough viewports.

Reply:
Done